core/src/main/java/org/jdbi/v3/GeneratedKeys.java

```diff
@@ -13,13 +13,14 @@
  */
 package org.jdbi.v3;
 
+import com.fasterxml.classmate.ResolvedType;
+import org.jdbi.v3.exceptions.ResultSetException;
+import org.jdbi.v3.tweak.ResultSetMapper;
+
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.sql.Statement;
 
-import org.jdbi.v3.exceptions.ResultSetException;
-import org.jdbi.v3.tweak.ResultSetMapper;
```

Comment:
lots of import shuffling, we should make sure our IDEs are configured with the same ordering...